.claude/hooks/__tests__/smoke.sh

```
@@ -0,0 +1,187 @@
+#   2. pre-commit.sh stdin parsing — no-op when command isn't really git commit
+#   3. pre-commit.sh — blocks when gates missing; allows when fresh; blocks when stale
+#   4. pre-pr.sh stdin parsing — no-op when command isn't really gh pr create
+#   5. pre-pr.sh — blocks without cross_model_review, allows with it
+set -euo pipefail
+
+ROOT=$(git rev-parse --show-toplevel)
+cd "$ROOT"
+
+HOOKS="$ROOT/.claude/hooks"
+# shellcheck source=../workflow-state.sh
+source "$HOOKS/workflow-state.sh"
```

Shellcheck (0.11.0)

[info] Not following: ../workflow-state.sh was not specified as input (see shellcheck -x).

(SC1091)